server/docs/source/index.html.md

```diff
@@ -428,7 +429,7 @@ Update a node details.
 
 ### Endpoint
 
-`PUT /v1/nodes/{node_id}`
+`PUT /v1/nodes/{id}`
```

Comment:
I think the "consistent" choice here would be `{node_id}`, but then that is super-confusing re `node['id']` vs `node['node_id']` - it needs to be the first one.